Userland/Libraries/LibVideo/VP9/Parser.cpp

```diff
@@ -18,6 +19,12 @@
 #    pragma GCC optimize("O3")
 #endif
 
+// Threading is unstable in Serenity with smp=on, but is stable in Lagom.
+// Let's keep this disabled on Serenity for now.
+#if !defined(AK_OS_SERENITY)
+#    define VP9_TILE_THREADING
+#endif
```

Comment:
I'd argue that exposing these bugs with SMP enabled will motivate more people to fix them :^) and SMP is disabled by default anyway.

Reply:
Fair enough, I think that the threading does incur a slight penalty with SMP off but it's probably fine. I'll be doing most of my testing in Lagom anyway :^)